pages/ssj/[workflow]/[phase]/[milestone]/index.js

```diff
@@ -2,14 +2,14 @@ import { useRouter } from "next/router";
 import { useState } from "react";
 import { styled, css } from "@mui/material/styles";
 import { useForm, Controller } from "react-hook-form";
-import { Container, Draggable } from "react-smooth-dnd";
+// import { Container, Draggable } from "react-smooth-dnd";
 import { arrayMoveImmutable } from "array-move";
 import getAuthHeader from "@lib/getAuthHeader";
 import processesApi from "@api/workflow/processes";
 import { clearLoggedInState, redirectLoginProps } from "@lib/handleLogout";
 import Skeleton from "@mui/material/Skeleton";
 
-import useAuth from "@lib/utils/useAuth";
+// import useAuth from "@lib/utils/useAuth";
```

Comment:
Are you sure you want to keep this commented out?

Reply:
THanks for catching, do not want to keep commented